### PR TITLE
[ICDS Dashboard] fix for ICDS-740; alert banner and side nav fix

### PR DESCRIPTION
**Pull request by dhiraj-beehyv**
[Jira Ticket](https://dimagi-dev.atlassian.net/browse/ICDS-740)

Fixed the ICDS dashboard view when alert is on and side navigation on window resize.

### CODE REVIEW

**Comment by Rohit25negi** [comment]
@dhiraj-beehyv  can you put the screen shot.

File: custom/icds_reports/static/js/directives/adhaar-beneficiary/adhaar-beneficiary.directive.js [hunk 1]
```diff
@@ -2,10 +2,11 @@
 var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function AdhaarController($scope, $routeParams, $location, $filter, demographicsService, locationsService,
-    userLocationId, storageService, haveAccessToAllLocations, baseControllersService) {
+    userLocationId, storageService, haveAccessToAllLocations, baseControllersService, isAlertActive) {
     baseControllersService.BaseController.call(this, $scope, $routeParams, $location, locationsService,
         userLocationId, storageService, haveAccessToAllLocations);
     var vm = this;
+    vm.isAlertActive = isAlertActive;
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/adhaar-beneficiary/adhaar-beneficiary.directive.js]: Where is this used?

File: custom/icds_reports/static/js/directives/system-usage/system-usage.directive.js [hunk 2]
```diff
@@ -2,19 +2,27 @@
...
     vm.selectedLocations = [];
+    vm.isAlertActive = isAlertActive;
 
     vm.prevDay = moment().subtract(1, 'days').format('Do MMMM, YYYY');
     vm.currentMonth = moment().format("MMMM");
     vm.lastDayOfPreviousMonth = moment().set('date', 1).subtract(1, 'days').format('Do MMMM, YYYY');
 
+
+
+    angular.element(document).ready(function () {
```
[the remark this answers is not in the document]
Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/system-usage/system-usage.directive.js]: @dhiraj-beehyv this looks concerning?

File: custom/icds_reports/static/js/directives/system-usage/system-usage.directive.js [hunk 3]
```diff
@@ -2,19 +2,27 @@
...
+       console.log("System Usage");
```
[the remark this answers is not in the document]
Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/system-usage/system-usage.directive.js]: @dhiraj-beehyv  please remove all console statements

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
@dhiraj-beehyv  can you put the screenshot?
Did you tested locally?
Also left a couple of comments

**Comment by dhiraj-beehyv** [comment]
> @dhiraj-beehyv can you put the screen shot.

alert fix
alert fix 2
alert fix3

File: custom/icds_reports/static/js/directives/adhaar-beneficiary/adhaar-beneficiary.directive.js (hunk 1 above)
Comment (dhiraj-beehyv) [reply custom/icds_reports/static/js/directives/adhaar-beneficiary/adhaar-beneficiary.directive.js]: It is used in the html templates to apply a style class for handling alert, example given below:

https://github.com/dimagi/commcare-hq/pull/25258/files#diff-9d5ea3110287b8e26b2d5cbac57e9333R2

File: custom/icds_reports/static/js/directives/system-usage/system-usage.directive.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (dhiraj-beehyv) [reply custom/icds_reports/static/js/directives/system-usage/system-usage.directive.js]: Removed the console statement.

**Comment by Rohit25negi** [comment]
@dhiraj-beehyv  also, does the banner works fine while scrolling up and down?

**Comment by Rohit25negi** [comment]
@dhiraj-beehyv  also the test case failure looks real

```
1) Adolescent Girls Directive "before each" hook: workFn for "tests instantiate the controller properly":
     Error: [$injector:unpr] http://errors.angularjs.org/1.4.4/$injector/unpr?p0=isAlertActiveProvider%20%3C-%20isAlertActive
      at http://localhost:8000/static/angular/angular.min.js:40
      at d (http://localhost:8000/static/angular/angular.min.js:38)
      at http://localhost:8000/static/angular/angular.min.js:40
      at d (http://localhost:8000/static/angular/angular.min.js:38)
      at e (http://localhost:8000/static/angular/angular.min.js:39)
      at http://localhost:8000/static/angular/angular.min.js:80
      at K (http://localhost:8000/static/angular/angular.min.js:61)
      at http://localhost:8000/static/angular/angular.min.js:68
      at http://localhost:8000/static/angular/angular.min.js:118
      at http://localhost:8000/static/angular/angular.min.js:133
      at http://localhost:8000/static/angular/angular.min.js:130
      at http://localhost:8000/static/angular/angular.min.js:133
      at g (http://localhost:8000/static/angular/angular.min.js:87)
      at handleResponse (http://localhost:8000/static/angular-mocks/angular-mocks.js:1158)
      at http://localhost:8000/static/angular-mocks/angular-mocks.js:1516
      at http://localhost:8000/static/icds_reports/js/spec/adolescent-girls.directive.spec.js:36
      at e (http://localhost:8000/static/angular/angular.min.js:39)
      at workFn (http://localhost:8000/static/angular-mocks/angular-mocks.js:2401)
  undefined```

**Comment by dhiraj-beehyv** [comment]
> @dhiraj-beehyv also, does the banner works fine while scrolling up and down?

Yes

**Comment by dhiraj-beehyv** [comment]
> @dhiraj-beehyv also the test case failure looks real
> 
> ```
> 1) Adolescent Girls Directive "before each" hook: workFn for "tests instantiate the controller properly":
>      Error: [$injector:unpr] http://errors.angularjs.org/1.4.4/$injector/unpr?p0=isAlertActiveProvider%20%3C-%20isAlertActive
>       at http://localhost:8000/static/angular/angular.min.js:40
>       at d (http://localhost:8000/static/angular/angular.min.js:38)
>       at http://localhost:8000/static/angular/angular.min.js:40
>       at d (http://localhost:8000/static/angular/angular.min.js:38)
>       at e (http://localhost:8000/static/angular/angular.min.js:39)
>       at http://localhost:8000/static/angular/angular.min.js:80
>       at K (http://localhost:8000/static/angular/angular.min.js:61)
>       at http://localhost:8000/static/angular/angular.min.js:68
>       at http://localhost:8000/static/angular/angular.min.js:118
>       at http://localhost:8000/static/angular/angular.min.js:133
>       at http://localhost:8000/static/angular/angular.min.js:130
>       at http://localhost:8000/static/angular/angular.min.js:133
>       at g (http://localhost:8000/static/angular/angular.min.js:87)
>       at handleResponse (http://localhost:8000/static/angular-mocks/angular-mocks.js:1158)
>       at http://localhost:8000/static/angular-mocks/angular-mocks.js:1516
>       at http://localhost:8000/static/icds_reports/js/spec/adolescent-girls.directive.spec.js:36
>       at e (http://localhost:8000/static/angular/angular.min.js:39)
>       at workFn (http://localhost:8000/static/angular-mocks/angular-mocks.js:2401)
>   undefined```
> ```

Fixed the test cases

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**